### PR TITLE
[plugin.service.gto] v2.0.23

### PR DESCRIPTION
**Pull request by b-jesch**
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Updates up from last 2.0.12:

- 2.0.23
minor fixes
- 2.0.22
some skin modifications
sports scraper added
- 2.0.18
add record capabilities
- 2.0.14
Some more properties, e.g. search titles in local database and flag the result

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.service.gto/addon.xml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.12" provider-name="Birger Jesch">
+<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.23" provider-name="Birger Jesch">
     <requires>
-        <import addon="xbmc.python" version="2.24.0"/>
+        <import addon="xbmc.python" version="2.25.0"/>
```
Comment (learningit) [inline plugin.service.gto/addon.xml]: Without reviewing further, xbmc.python = 2.25.0 is for Krypton.
Your PR is against the Jarvis branch.

Comment (b-jesch) [reply plugin.service.gto/addon.xml]: I use indeed some new JSON-RPC introduced with Krypton (PVR capabiltities). So I don't know what is the best way to bump the addon to Krypton, so excuse me.
 - should I discard/reject the PR here and mark the addon as broken/deprecated?
- Make a new PR on Krypton?
- Or can you suggest me another way to continue development only on krypton (and maybe later versions). What is the usual procedure here?

I use PyCharm as my preferred IDE which has some Github capabilities.

Comment (learningit) [reply plugin.service.gto/addon.xml]: This is a good question.

If you want to only support a Krypton version and force your current users to upgrade:

1) Please note that you shouldn't/can't submit a PR with xbmc.python set to a value that corresponds to a Kodi version higher than the branch you're submitting to. 

2) Submit a PR on the Jarvis branch setting the addon.xml field \<broken\>\</broken\>. You can include a message between to let them know there's a Krypton version available, e.g. \<broken\>Please upgrade to Krypton to continue using this add-on\</broken\>. You can actually delete all files except addon.xml, if you wish.

3) Then submit a PR on the Krypton branch with xbmc.python set to 2.25.0 for the new (higher) version of the add-on that you want to support.

Comment (b-jesch) [reply plugin.service.gto/addon.xml]: Thank you very much for your explanation. I'll do the necessary steps at monday, my dev system is @work.

**Comment by learningit** [comment]
This looks fine. I'll merge after the Krypton update is in place.

**Comment by learningit** [comment]
Closed until a solution for the Krypton version is found.

**Pull request closed without merging**